### DISCUSSION TITLE
Cors module recipe example

### DISCUSSION THREAD

**Discussion by vitalybrandes** [OrchardCMS/OrchardCore#14161] [Q&A]
Is there any Cors module recipe example i can use?

**Comment by Piedone** [comment]
You can configure the module from the admin and export the settings to see how they look like in a recipe. Then when you get a hang of it, you'll be able to just edit the recipe directly.

**Reply by hishamco** [reply]
Unfortunately, there's deployment for CORS

**Reply by Piedone** [reply]
Do you mean there's NO deployment?

**Reply by hishamco** [reply]
Yes, I didn't see any deployment

**Reply by vitalybrandes** [reply]
@Piedone 
there is no option to export deployments steps for cors :-(
![image](https://github.com/OrchardCMS/OrchardCore/assets/61165619/323d823d-52f5-48e7-a405-af3c0f30d78f)

**Reply by hishamco** [reply]
I just created an issue https://github.com/OrchardCMS/OrchardCore/issues/14224 for tracking this item
